projects/packages/forms/src/blocks/field-consent/index.js

```diff
@@ -50,7 +50,7 @@ const settings = {
 	},
 	providesContext: {
 		...defaultSettings.providesContext,
-		'jetpack/field-defaultValue': 'defaultValue',
+		'jetpack/field-default-value': 'defaultValue',
```

Comment:
I kept this, but I can't see where the value is used in the consent block.

Probably not worth changing it right now.